### PR TITLE
Aida 1959

### PR DESCRIPTION
**Pull request by phile-caci**
@gitnordt ,

K.. I think it's done.  tested against our test data and BBN's TA1, looks about right.  I didn't do test cases as this has gotten very time consuming.  I'll create another ticket to handle the test cases.  But it should be enough for teams to start looking at.

Also, I'm not sure why we duplicate things between dwd_aif.shacl and aida_ontology.shacl, I don't think we need to.  I'll have to review this at a later date.  Wanted to get something working first.

LMK if you see anything crazy.

Thanks!
Phi

### CODE REVIEW

File: java/src/main/java/com/ncc/aif/ValidateAIF.java [hunk 1]
```diff
@@ -59,6 +59,7 @@
     private static final String AIDA_SHACL_RESNAME = AIF_ROOT + "aida_ontology.shacl";
     private static final String NIST_SHACL_RESNAME = AIF_ROOT + "restricted_aif.shacl";
     private static final String NIST_HYPOTHESIS_SHACL_RESNAME = AIF_ROOT + "restricted_hypothesis_aif.shacl";
```
Comment (gitnordt) [inline java/src/main/java/com/ncc/aif/ValidateAIF.java:61]: Is this hypothesis variable still being used?

File: java/src/main/java/com/ncc/aif/ValidateAIF.java [hunk 2]
```diff
@@ -71,6 +72,7 @@
     private static Model shaclModel;
     private static Model nistModel;
     private static Model nistHypoModel;
```
Comment (gitnordt) [inline java/src/main/java/com/ncc/aif/ValidateAIF.java:74]: Is this hypothesis mode still being used?

File: java/src/main/java/com/ncc/aif/ValidateAIF.java [hunk 3]
```diff
@@ -184,7 +192,8 @@ public static ValidateAIF createForDWD(Restriction restriction) {
         restrictions.add("com/ncc/aif/dwd_aif.shacl");
         switch (restriction) {
             case NIST_TA3:
-                restrictions.add(NIST_HYPOTHESIS_SHACL_RESNAME);
+                restrictions.add(NIST_CLAIMFRAME_SHACL_RESNAME);
+                //restrictions.add(NIST_HYPOTHESIS_SHACL_RESNAME);
```
Comment (gitnordt) [inline java/src/main/java/com/ncc/aif/ValidateAIF.java:196]: Should we comment out or erase references to hypothesis?

File: java/src/main/java/com/ncc/aif/ont2javagen/OntologyGeneration.java [hunk 4]
```diff
@@ -86,7 +86,8 @@ public static void main(String[] args) {
             getStream.get().forEach(OntologyGeneration::writePythonFile);
             writeShaclJavaFile(aifRoot + "aida_ontology.shacl",
                     aifRoot + "restricted_aif.shacl",
-                    aifRoot + "restricted_hypothesis_aif.shacl");
+                    aifRoot + "restricted_hypothesis_aif.shacl",
```
Comment (gitnordt) [inline java/src/main/java/com/ncc/aif/ont2javagen/OntologyGeneration.java:89]: Is this hypothesis shacl file still needed?

File: java/src/test/java/com/ncc/aif/TestUtils.java [hunk 5]
```diff
@@ -359,6 +359,8 @@ void testValid(String testDescription) {
      * Assert that the test with the specified description is valid based on the current model, the supplied hypothesis,
      * and the current validator.
      */
+
+
     void testValidWithHypothesis(String testDescription, @Nullable Model hypothesisModel) {
         assertAndDumpWithHypothesis(testDescription, true, hypothesisModel);
```
Comment (gitnordt) [inline java/src/test/java/com/ncc/aif/TestUtils.java:365]: Do we still need to test the hypothesis model?

File: java/src/main/java/com/ncc/aif/ValidateAIF.java (hunk 3 above)
Comment (phile-caci) [reply java/src/main/java/com/ncc/aif/ValidateAIF.java:196]: removed

File: java/src/main/java/com/ncc/aif/ValidateAIF.java (hunk 1 above)
Comment (phile-caci) [reply java/src/main/java/com/ncc/aif/ValidateAIF.java:61]: removed

File: java/src/main/java/com/ncc/aif/ValidateAIF.java (hunk 2 above)
Comment (phile-caci) [reply java/src/main/java/com/ncc/aif/ValidateAIF.java:74]: removed

File: java/src/test/java/com/ncc/aif/TestUtils.java (hunk 5 above)
Comment (phile-caci) [reply java/src/test/java/com/ncc/aif/TestUtils.java]: K.. I removed all Hypo testing.  There are no claimframe tests yet but I have a ticket to handle that next sprint.  LMK if there's anything else.

**Review by gitnordt — APPROVED** [review]

**Pull request merged**